web/src/projects/createProject.ts

```diff
@@ -31,11 +31,12 @@ export async function internalCreateProject(
   agentAddress: AgentPubKeyB64,
   dispatch: any,
   iInstallProject: typeof installProject,
+  _finalizeCreateProject: typeof finalizeCreateProject,
```

Comment:
I was experimenting with writing these with `i` prefix instead of the `_` just because the _ is also used as a type hint to ignore unused variables and stuff... so it might hurt maintainability to use them. thoughts?

Reply:
ah yeah i had seen that. that is a good point. i can go through and change them all to `i` prefixes

Reply:
Ok cool.